app/controllers/static_pages_controller.rb

```diff
@@ -1,5 +1,9 @@
 class StaticPagesController < ApplicationController
-  def home; end
+  def home
+    @products = Product.page(params[:page]).per(Settings.page_record_medium)
+                       .recent
+    @categorys = Category.select(:name)
```

Comment:
```suggestion
    @products = Product.page(params[:page]).per(Settings.page_record_medium)
                       .recent
    @categorys = Category.select(:name)
```
code này lặp nè, em tạo 1 method rồi dùng chung dc em

Reply:
Dạ, chỗ này bữa em viết thừa mà chưa xóa ạ